frontend/public/components/utils/dropdown.jsx

```diff
@@ -447,7 +447,8 @@ export class ContainerDropdown extends React.PureComponent {
       items={dropdownItems}
       spacerBefore={spacerBefore}
       title={title}
-      onChange={onChange} />;
+      onChange={onChange}
+      noSelection={!_.isEmpty(currentKey)} />;
```

Comment:
This looks wrong... `noSelection` is meant to be used for action menus where an item isn't selected. In this dropdown, and item should be selected.